app/assets/javascripts/angular/components/organizationSignatures/organizationSignatures.html

```diff
@@ -0,0 +1,75 @@
+                <button class='f5 ml1 pv3 ph4 b--pivot-gray-darker bg-pivot-gray-darker pivot-white br2 search dim' ng-click="$ctrl.search(search, $ctrl.tableState)"></button>
+            </form>
+        </span>
+    </div>
+    <table st-table="$ctrl.signatures" st-pipe="$ctrl.load" st-safe-src="$ctrl.signatureData" class="grid w-100">
+        <thead>
+            <tr class='bg-pivot-dark-blue'>
+                <th class='pa3 pivot-white br2 tc br--top br--left ttu' st-sort="updated_at">{{ 'signatures.grid.dateSigned' | t }}</th>
+                <th class='pa3 pivot-white ttu tc' st-sort="first_name">{{ 'signatures.grid.firstName' | t }}</th>
+                <th class='pa3 pivot-white ttu tc' st-sort="last_name">{{ 'signatures.grid.lastName' | t }}</th>
+                <th class='pa3 pivot-white ttu tc' st-sort="code_of_conduct">{{ 'signatures.grid.codeOfConduct' | t }}</th>
+                <th class='pa3 pivot-white br2 ttu tc br--top br--right' st-sort="statement_of_faith">{{ 'signatures.grid.statementOfFaith' | t }}</th>
+            </tr>
+        </thead>
+        <tbody class='grid-content'>
+            <tr ng-repeat="row in $ctrl.signatures">
+                <td class='pa3 tc'>{{row.updated_at | date}}</td>
+                <td class='pa3 tc'>{{row.first_name}}</td>
+                <td class='pa3 tc'>{{row.last_name}}</td>
+                <td class='pa3 tc'>
+                    <span ng-if="row.code_of_conduct === 'accepted'">{{ 'signatures.grid.accepted' | t }}</span>
+                    <span ng-if="row.code_of_conduct === 'declined'">{{ 'signatures.grid.declined' | t }}</span>
+                    <span ng-if="!row.code_of_conduct">{{ 'signatures.grid.notAccecptedYet' | t }}</span>
+                </td>
+                <td class='pa3 tc'>
+                    <span ng-if="row.statement_of_faith === 'accepted'">{{ 'signatures.grid.accepted' | t }}</span>
+                    <span ng-if="row.statement_of_faith === 'declined'">{{ 'signatures.grid.declined' | t }}</span>
+                    <span ng-if="!row.statement_of_faith">{{ 'signatures.grid.notAccecptedYet' | t }}</span>
+                </td>
+            </tr>
+            <tr ng-if="$ctrl.signatures.length <=0">
+                <td class='pa3 tc' colspan="5" ng-if="$ctrl.isLoading">{{ 'signatures.grid.loading' | t }}</td>
+                <td class='pa3 tc' colspan="5" ng-if="!$ctrl.isLoading">{{ 'signatures.grid.notFound' | t }}</td>
+            </tr>
+        </tbody>
+        <tfoot>
+			<tr>
+				<td colspan="5" class="text-center">
+                    <div class="pagination mt3" ng-if="$ctrl.tableState.pagination.pages.length > 1">
+                        <ul class="list flex">
+                            <li ng-if="$ctrl.tableState.pagination.currentPage > 1">
+                                <a href ng-click="$ctrl.previousPage(search);" class='f5 pv1 ph2 pivot-gray-darker br2 dim'> < </a>
```

Comment:
```suggestion
                                <a href ng-click="$ctrl.previousPage(search);" class='f5 pv1 ph2 pivot-gray-darker br2 dim'> &lt; </a>
```
I think this is killing Webpack's html-loader parser when running on Netlify.